examples/parametrized/diffusion3D_multigpucpu_novis.jl

```diff
@@ -0,0 +1,69 @@
+const USE_GPU = true
+using ImplicitGlobalGrid
+import MPI
+using ParallelStencil
+using ParallelStencil.FiniteDifferences3D
+@static if USE_GPU
+    @init_parallel_stencil(package=CUDA, ndims=3);
+else
+    @init_parallel_stencil(package=Threads, ndims=3);
+end
+
+@parallel function diffusion3D_step!(T2::Data.Array{_T}, T::Data.Array{_T}, Ci::Data.Array{_T}, lam::_T, dt::_T, _dx::_T, _dy::_T, _dz::_T) where _T <: ParallelStencil.PSNumber
```

Comment:
Is `where _T <: ParallelStencil.PSNumber` mandatory with parametrised number type ?

Reply:
No, this is not mandatory, but recommended to avoid unexpected errors.